### DISCUSSION TITLE
[RFC Call] Agenda - January 18, 2021

### DISCUSSION THREAD

**Discussion by jonathantneal** [withastro/roadmap#71] [Proposal]
# Agenda

* @FredKSchott - collect feedback - https://github.com/withastro/rfcs/discussions/78
* everyone - discussion - Community Call

--- 

_If there is something that you would like to see discussed, please comment below. We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests.](https://github.com/withastro/rfcs/discussions)_

**Comment by FredKSchott** [comment]
If we have nothing else, I would like to spend a few minutes gathering feedback on this: https://github.com/withastro/rfcs/discussions/78

I could also give a quick follow-up update on the discussions from last week, and next steps based on the feedback I got.

**Reply by jonathantneal** [reply]
Added.

**Comment by FredKSchott** [comment]
If we have time, I would also love to hear feedback on this call from the community! What do people like about it, what could be improved? I've been thinking about how we could make `RFC Call -> Community Call` and experiment with other agenda items (not just RFC discussions). Curious to hear thoughts!

**Reply by aFuzzyBear** [reply]
I have a suggestion for one on that end, but I feel like its a bit premature atm, idk, 
I suggest that we have a similar setup for the documentation and a team organised along those lines, I can elaborate more if you wish...

**Reply by jonathantneal** [reply]
+1 on talking about that. I also plan to add a note at the beginning about how long our call it, to help separate when the call ends and the so-far-informal 'community after hours' that tend to follow.
